src/purify.js

```diff
@@ -7,9 +7,8 @@
         define(function(){ return factory(root); });
     } else if (typeof module !== 'undefined') {
         module.exports = factory(root);
-    } else {
-        root.DOMPurify = factory(root);
     }
+    root.DOMPurify = factory(root);
```

Comment:
We should not interact with the `factory` of the surrounding UMD build in the first place. This has changed after merging #206 where we export a function `createDOMPurify` function you can use to wrap the library onto another global. You can see an example in our test setup for jsdom [here](https://github.com/cure53/DOMPurify/blob/master/test/jsdom-node.js#L6).